### ISSUE TITLE
height not respected in iOS 11 GM

### ISSUE THREAD

**Issue by tbatzakas** [maxep/MXParallaxHeader#81]
While everything works perfectly in iOS 10, in iOS 11 with UITableViewController the headerView begins with the wrong size. It appears that the size is the initial size + 20 px (status bar?) + 44px (navigationBar?). If the navigationBar and status bar is removed programmatically, it works as it should.

**Comment by tbatzakas** [comment]
If anyone else has this problem, please refer to this thread https://github.com/maxep/MXParallaxHeader/pull/78/files/6e9936b26844654b72a547100d256f301068a408

**Comment by kristiyandobrev** [comment]
I don't know but it might be a similar problem that I have experienced only on `iOS 11`.  Here is the case:

When starting my application the `scrollView` looks ok. So next without scrolling anything I am opening another `ViewController` and then going back to the one that has the `scollView`. The result is that the `scrollView`'s  content inset changed without me scrolling it .... !? 

According to the [docs](https://developer.apple.com/documentation/uikit/uiscrollview/2902261-contentinsetadjustmentbehavior) there is a new property called `contentInsetAdjustmentBehavior` which default value is `.automatic`, so for my case I simply changed that value to `.never` and I got my old behaviour back.
